### ISSUE TITLE
Main site pricing page: colour contrast issues [a11y]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [supabase/supabase#15357]
## Describe the bug

There are a few places where there is insufficient colour contrast between the foreground text and the background on the [pricing page](https://github.com/supabase/supabase/blob/master/apps/www/pages/pricing/index.tsx) on the main website.

**note:** The below are issues for *dark mode*.

## Expected behaviour

For small text this colour contrast ratio should be a minimum of 4.5:1.

There are several places where the contrast is not high enough (when viewing in dark mode):

### "Most Popular"
The "most popular" surround for the pro pricing tier is only 1.99:1

!["Most Popular" on pro tier highlight](https://github.com/supabase/supabase/assets/121672086/374c0e12-3eba-4bc7-b589-6c824a74d6a9)

### Pricing table
The "/month/project" and "/From" on the price section is only 3.17:1 contrast ratio.
 
![Pricing section on each column in the pricing table](https://github.com/supabase/supabase/assets/121672086/6e1aeb8d-a797-4f06-a705-a55afeeeb5ea)

## Additional context

I only checked the pricing tables, there may be other colour contrast issues on the page.

Again, for clarity, the minimum contrast should be 4.5:1, colours can be checked with a colour contrast checker such as https://webaim.org/resources/contrastchecker/

This is related to [WCAG Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Comment by sambhavgupta0705** [comment]
I would like to work on this issue.

**Comment by MildTomato** [comment]
Yep, this checks out @GrahamTheDevRel - thanks for the roasting 💯 

I think we mistakenly used "radix-9" colors assuming they would be AAA pass on all backgrounds.
I think I may have misremembered some very old docs, as now "11" is deemed as safe contrast, when i remember being "9".
https://www.radix-ui.com/docs/colors/tests/contrast

As mentioned in [an earlier comment](https://github.com/supabase/supabase/issues/15356#issuecomment-1607950218) we are putting some real resources into improving our UI and a11y so we need to solve this, and maintain it across all our apps.

Screenshot below of the failed contrast checker test.

<img width="846" alt="Screenshot 2023-06-27 at 1 56 02 AM" src="https://github.com/supabase/supabase/assets/8291514/fa790871-c4e3-4b37-bf7a-c129eee7d8d7">

**Comment by burggraf** [comment]
@saltcod Has this been addressed yet?

**Comment by saltcod** [comment]
Looks like recent changes to our colours setup have fixed this. 
![screenshot-2024-01-18-at-08 52 53](https://github.com/supabase/supabase/assets/105593/87eddeb0-6457-45a0-a6a4-ed6d43342548)

Marking as closed, but feel free to reopen if I've missed something.
